core/templates/dev/head/pages/signup/licence_explanation_modal_directive.html

```diff
@@ -6,7 +6,7 @@ <h3 translate="I18N_SIGNUP_WHY_LICENSE"></h3>
   <p style="line-height: 1.5;" translate="I18N_SIGNUP_SITE_OBJECTIVE" translate-values="{sitename: siteName}">
   </p>
 
-  <p style="line-height: 1.5;" translate="I18N_SIGNUP_LICENSE_OBJECTIVE" translate-licenselink="<a href=&quot;https://creativecommons.org/licenses/by-sa/4.0/&quot; target=&quot;_blank&quot;>CC-BY-SA v4.0</a>">
+  <p style="line-height: 1.5;" translate="I18N_SIGNUP_LICENSE_OBJECTIVE" translate-licenselink="<a href=&quot;https://creativecommons.org/licenses/by-sa/4.0/&quot; target=&quot;_blank&quot;</a>">
```

Comment:
Looks like the CC-BY-SA v4.0 disappeared

Reply:
Oops, reverted back!